tests/unit/Parameters/HooksCreateParametersTest.php

```diff
@@ -30,7 +30,7 @@ public function testHooksCreateParameters()
 
         // Test setters that are ignored by the constructor
         $hooksCreateParameters->setMeetingId($meetingId = $this->faker->uuid);
-        $hooksCreateParameters->setGetRaw($getRaw = $this->faker->boolean);
+        $hooksCreateParameters->setGetRaw($getRaw       = $this->faker->boolean);
```

Comment:
I don't like this style.
@sualko @FelixJacobi Any ideas how to keep the old formatting?

Reply:
It's probably related to https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/master/doc/rules/operator/binary_operator_spaces.rst, but I think we can not disable it inside method calls.

Reply:
Hm, maybe we should move to PSR-12 as a well defined and accepted code style and stop using the space intention at all. I personally really don't like it, as it is a waste of bytes and produces big diffs when adding a new parameter is added and so on, because the whole list must be reformatted:

~~~diff
-$fooBar = 'foo',
-$barBaz = 'bar',
+$fooBar                         = 'foo',
+$barBaz                         = 'bar',
+$veryLongParameter = null
~~~

To illustrate what I mean. This code style enforces such reformats and this is bad in my opinion on reading of diffs. I would stop using that.

Reply:
@sualko @FelixJacobi I updated this branch to the new tooling solution and changed to symfony style. 
To prevent the yoda style and allowing multi line throws I added two custom rules.
What do you think about the new code style?